src/OSPSuite.Infrastructure.Import/Core/DataSheetCollection.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using OSPSuite.Utility.Collections;
+
+namespace OSPSuite.Infrastructure.Import.Core
+{
+   /// <summary>
+   ///    a collection of excel dataSheet sheets
+   /// </summary>
+   public class DataSheetCollection : IEnumerable<DataSheet>
```

Comment:
Why do we have a IDataSheet and a DataSheet? Do we need the interface ? if yes, this should be a collection of IDataSheet probably. It not remove

Reply:
interface removed